onyo/lib/commands.py

```diff
@@ -293,15 +293,16 @@ def _edit_asset(inventory: Inventory,
     if not editor:
         editor = inventory.repo.get_editor()
 
-    # Store original reserved keys of `asset`, in order to re-assign
+    # Store original pseudo-keys of `asset`, in order to re-assign
     # them when loading edited file from disc. This is relevant, when
-    # `operation` uses them (`Inventory.add_asset`)
-    reserved_keys = {k: v for k, v in asset.items() if k in RESERVED_KEYS}
-    disallowed_keys = RESERVED_KEYS + PSEUDO_KEYS
-    disallowed_keys.remove("is_asset_directory")
+    # `operation` uses them (`Inventory.add_asset/modify_asset`).
+    reserved_keys = {k: v for k, v in tmp_asset.items() if k in list(PSEUDO_KEYS.keys()) + ['template']}
```

Comment:
True. Missed that spot. Will fix in this PR.

Reply:
Actually, no. Not needed, because the incoming dict is supposed to be an `Item`. Hence, aliases used to create it in the first place were already resolved.
The "supposed to be an Item" is key, though. Happens to be the bug you saw in the demo-repo log ;-)